tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/javainterop/basic/FieldAccessMutateTest.java

```diff
@@ -80,6 +80,16 @@ public void testStaticPrimitiveFieldMutate() {
         Assert.assertEquals(JavaFieldAccessMutate.aShort, 345);
     }
 
+    @Test(description = "Test static boolean field mutate")
+    public void testStaticPrimitiveFieldMutate2() {
+        BRunUtil.invoke(result, "testStaticPrimitiveFieldMutate2");
```

Comment:
Shall we have a better name that gives the meaning rather than have a `2` in the method name? Probably like `testStaticBooleanFieldMutate`

Reply:
I named it as `testStaticPrimitiveFieldMutate2` because already a test is available for primitive types static field. I thought it would be better to name it as 2nd test for static primitive field mutate rather than specially mention it as boolean test. That's why I gave it in the description. Furthermore, I've seen, this kind of naming habit has been used in some other tests also. WDYT?

Reply:
If we could be more specific that is better IMHO. Numbering is used when we cannot be more specific. It is usually better to avoid numbers in names if we could help it.

Reply:
Sure. resolved